### DISCUSSION TITLE
Model theads value

### DISCUSSION THREAD

**Discussion by llex333** [coin-or/python-mip#339] [Q&A]
Hi, community.

I try to understand how threads value of [model](https://github.com/coin-or/python-mip/blob/master/mip/model.py) working. i do several tests and see no difference between values -1,0 or others. If python-mip use Cbc solver, does it mean, that it can use not only one core for solving problem? Cause from side i see that using only one core, when i have much more is not optimal.

**Comment by sebheger** [comment]
See #311 and #253. There are several issues around that problem.
